prime-angular-frontend/src/app/modules/adjudication/pages/enrolments/enrolments.component.ts

```diff
@@ -73,10 +75,6 @@ export class EnrolmentsComponent implements OnInit {
     return (currentStatusCode === EnrolmentStatus.UNDER_REVIEW);
   }
 
-  public isSuperAdmin(): boolean {
```

Comment:
ERROR in src/app/modules/adjudication/pages/enrolments/enrolments.component.html(178,15): Property 'isSuperAdmin' does not exist on 

this is till used on the template i believe

Reply:
Whoops, this is why I'm supposed to build prod after changes